src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Setter.cs

```diff
@@ -24,6 +24,7 @@ namespace System.Windows
     /// </summary>
     [XamlSetMarkupExtensionAttribute("ReceiveMarkupExtension")]
     [XamlSetTypeConverterAttribute("ReceiveTypeConverter")] 
+    [ContentProperty("Value")]
```

Comment:
Nitpicking: you might want to use `nameof` here.

Reply:
Yes, thought about using nameof, but used on purpose a string for now, as it's like this in all other places I saw. I don't know if it's a good idea. But I guess we should be able to use a Roslyn analyzer later to get rid of all hardcoded strings and use for example nameof on every ContentPropertyAttribute in the whole WPF source code.